presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -118,10 +122,10 @@ public void testBigintSimpleTypedSet()
                         createLongsBlock(null, null, null),
                         createLongSequenceBlock(0, 100),
                         createLongSequenceBlock(-100, 100),
-                        createLongsBlock(Collections.nCopies(1, null)),
-                        createLongsBlock(Collections.nCopies(100, null)),
-                        createLongsBlock(Collections.nCopies(expectedSetSizes.get(expectedSetSizes.size() - 1) * 2, null)),
-                        createLongsBlock(Collections.nCopies(expectedSetSizes.get(expectedSetSizes.size() - 1) * 2, 0L)));
```

Comment:
@electrum 
i did add the static imports. 
but these weren't lines of code I added, FYI (fine if I'm just cleaning up old kruft, i just want to make sure I understand)